x-pack/plugins/security_solution/public/common/containers/use_full_screen/index.tsx

```diff
@@ -29,6 +29,25 @@ export const resetScroll = () => {
   }, 0);
 };
 
+export const setDataGridFullScreen = (isExpanded: boolean) => {
+  const clickEvent = new KeyboardEvent('click', {
```

Comment:
oof

Reply:
can you please put a `// TODO` with a link to the Eui issue so we remember to come back and refactor once a method is available. @chandlerprall could you please help us find an eui issue?

Reply:
https://github.com/elastic/eui/issues/5126